db/dcrpg/pgblockchain.go

```diff
@@ -4827,6 +4827,19 @@ func (pgb *ChainDB) GetAPITransaction(txid *chainhash.Hash) *apitypes.Tx {
 		return nil
 	}
 
+	msgTx, err := txhelpers.MsgTxFromHex(txraw.Hex)
```

Comment:
Oh now I remember why I didn't do this.  Expensive.  Oh well.

Reply:
Is there any thing you'd suggest to work around this?

Reply:
Nothing comes to mind, other than modifying the dcrd RPC, or querying the db (even slower).
This is fine